features/performance/assets/pre-boot-tuning.sh

```diff
@@ -68,7 +68,19 @@ else
     RHCOS_OSTREE_PATH=$(ls -td /boot/ostree/*/ | head -1)
     cp iso_initrd.img $RHCOS_OSTREE_PATH
     RHCOS_OSTREE_PATH=${RHCOS_OSTREE_PATH#"/boot"}
-    sed -i "s^initrd .*\$^& ${RHCOS_OSTREE_PATH}iso_initrd.img^" /boot/loader/entries/ostree-2-rhcos.conf
+
+    # Get current ostree config file
+    current_ver=1
+    entry_file=""
+    egrep $(uname -r) -lr /boot/loader/entries/ | while read -r line ; do
+        ver=`awk '/version/ {print $2}' $line`
+        if [ "$ver" -gt "$current_ver" ]; then
+           current_ver=$ver
+           entry_file=$line
+        fi
+    done
```

Comment:
nicer indeed, could you please add a couple lines to comment on line 72 to outline the logic?